tests/foreman/api/test_host.py

```diff
@@ -1068,7 +1066,7 @@ def test_positive_read_enc_information(
     host_enc_parameters = host_enc_info['data']['parameters']
     assert host_enc_parameters['organization'] == module_org.name
     assert host_enc_parameters['location'] == module_location.name
-    assert host_enc_parameters['content_view'] == module_cv_with_puppet_module.name
+    assert host_enc_parameters['content_view'] == default_contentview.name.replace(' ', '_')
```

Comment:
Can you elaborate the need for `.replace(' ', '_')`?  Just need clarification, that's all.

Reply:
enc parameters are saved this way, they use `_` underscore when there is multiple words

It was not problem before because contentview was created  inside `publish_puppet_module` with one word - 'alpha' of length from 6 to 12, but now it's Default Content view  which needts to be transformed to 'Default_Content_view'

Reply:
Why won't you use the `default_contentview.label` instead?